core/src/main/java/io/questdb/cairo/wal/seq/TableTransactionLogV2.java

```diff
@@ -483,7 +483,9 @@ public void toTop() {
 
         private void closePart() {
             if (partFd > -1) {
-                ff.munmap(address, partMapSize, MemoryTag.MMAP_TX_LOG_CURSOR);
+                if (address > 0) {
```

Comment:
nit: incosistent with similar checks
```suggestion
                if (address != 0) {
```